### PR TITLE
Add sUSD to liquidity and allow to modify the list using env vars

### PR DESCRIPTION
**Pull request by anxolin**
* Adds sUSD
* Adds an env var to easily mantain this list

### CODE REVIEW

File: src/const.ts [hunk 1]
```diff
@@ -102,7 +102,8 @@ export const GP_ORDER_TX_HASHES = {
   4: 'GP_ORDER_TX_HASHES_4',
 }
 
-export const LIQUIDITY_TOKEN_LIST = new Set(['USDT', 'TUSD', 'USDC', 'PAX', 'GUSD', 'DAI'])
+const LIQUIDITY_TOKEN_LIST_VALUES = process.env.LIQUIDITY_TOKEN_LIST || 'USDT,TUSD,USDC,PAX,GUSD,DAI,sUSD'
+export const LIQUIDITY_TOKEN_LIST = new Set(LIQUIDITY_TOKEN_LIST_VALUES.split(',').map(symbol => symbol.trim()))
```
Comment (Velenir) [inline src/const.ts:106]: `trim` just in case?

Comment (Velenir) [reply src/const.ts:106]: that makes me think, can there be token symbols with a space?
yes there can
but probably not starting or ending with (trimmable) space, that would be ridiculous

Comment (anxolin) [reply src/const.ts:106]: yes, because anyone can add a comma separated list of tokens. So in case there's a space somewhere, it won't break it

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr928--dexreact.review.gnosisdev.com

File: src/const.ts (hunk 1 above)
Comment (Velenir) [reply src/const.ts:106]: what if there's a token with comma in symbol, `T,KEN`, the horror :scream:

Comment (anxolin) [reply src/const.ts:106]: What if the token is a whitespace??

Comment (alfetopito) [reply src/const.ts:106]: >What if the token is a whitespace?? 
Doesn't deserve to be listed.

**Pull request merged**